app/services/DataAccess.scala

```diff
@@ -0,0 +1,13 @@
+package services
+
+import java.time.YearMonth
+import models._
+
+import scala.util.Try
+
+/**
+ * Created by chiua on 24/10/2017.
+ */
+trait DataAccess {
```

Comment:
If we know that only HBASE rest is going to be used, you could just use HBaseData rather than extending a trait.